File: Numbers/Row_Sum_Odd_Numbers/row_sum_odd_numbers.go
package main

import (
	"fmt"
	"math"
)

func main() {
	fmt.Println(rowSumOddNumbers(3) == 27)
	fmt.Println(rowSumOddNumbers(4) == 64)
	fmt.Println(rowSumOddNumbers(5) == 125)
}

func rowSumOddNumbers(rowNum int) int {
	return int(math.Pow(float64(rowNum), 3))
}

File: Numbers/Row_Sum_Odd_Numbers/RowSumOddNumbers.java
package Numbers.Row_Sum_Odd_Numbers;

public class RowSumOddNumbers {

    public static void main(String... args) {
        System.out.println(rowSumOddNumbers(3) == 27);
        System.out.println(rowSumOddNumbers(4) == 64);
        System.out.println(rowSumOddNumbers(5) == 125);
    }

    public static int rowSumOddNumbers(int n) {
        return (int) Math.pow(n, 3);
    }

}